webhook-service/lib/ip_resolver.go

```diff
@@ -0,0 +1,42 @@
+package lib
+
+import (
+	"net"
+	"net/url"
+
+	logger "github.com/sirupsen/logrus"
+)
+
+type IIPResolver interface {
```

Comment:
to be consistent with other interfaces, I suggest to also use `moq` to generate the mock implementations here

Reply:
Maybe we can have a discussion about that somewhen :)
For me, I've discovered that I am less and less making use of `moq` and co. But just try to have small interfaces (which this one clearly is) and implement the mock on the fly. I only use `moq` for bigger third party interfaces, actually

Reply:
same opinion on my side, for a few line mock, we do not need to generate it